### ISSUE TITLE
Could not find a version that satisfies the requirement torch==1.12.1+cu116

### ISSUE THREAD

**Issue by XMUykyz** [Akegarasu/lora-scripts#324]
bash install.bash
Creating python venv...
active venv
Installing torch & xformers...
Cuda Version:11.7
install torch 1.12.1+cu116
Looking in indexes: https://mirrors.aliyun.com/pypi/simple, https://download.pytorch.org/whl/cu116
ERROR: Could not find a version that satisfies the requirement torch==1.12.1+cu116 (from versions: 1.13.0, 1.13.0+cu116, 1.13.1, 1.13.1+cu116, 2.0.0, 2.0.1, 2.1.0, 2.1.1, 2.1.2)
ERROR: No matching distribution found for torch==1.12.1+cu116

**Comment by XMUykyz** [comment]
![image](https://github.com/Akegarasu/lora-scripts/assets/139086807/b1139e43-d1c3-444d-9144-a7de859ca964)
I changed the version from torch==1.12.1+cu116 to torch==1.13.1+cu116. The installation has started normally, I hope there is no problem.

**Comment by XMUykyz** [comment]
I see, because torch 1.12.1+cu116 does not have cp311-cp311 version, torchvision cu116 does not have cp311-cp311 version of the.whL file provided. Therefore, I tried to change the version of cu117 that meets cp311-cp311, and it could be installed, but an error was reported that it was a version conflict. I tried torchvision==0.15.0+cu117. And 0.15.1+cu117 and 0.15.2+cu117. Both tried and failed.
Error ：
ERROR:  Cannot install torch==1.13.1+cu117 and torchvision==0.15.0+cu117 because these package versions have conflicting dependencies.
![image](https://github.com/Akegarasu/lora-scripts/assets/139086807/a7c00008-21ae-4dea-a376-1c0a9b2b24b5)
